symfony/plugins/orangehrmCorePlugin/config/CorePluginConfiguration.php

```diff
@@ -65,6 +66,8 @@ public function initialize(Request $request): void
             'cookie_secure' => $isSecure,
             'cookie_httponly' => true,
             'cookie_path' => $path,
+            'cookie_samesite' => 'Strict',
+            'cookie_lifetime' => Config::MAX_IDLE_TIME
```

Comment:
Did you test the application by reducing the `Config::MAX_IDLE_TIME` time?

Reply:
yes cookie disappears and when reloading goes to login page and new cookie gets created

Reply:
Note: Cookie lifetime was removed from `CorePluginConfiguration` since it kicks out the user after half an hour regardless of whether they were idle or not